core/src/main/resources/application.conf

```diff
@@ -118,6 +120,11 @@ call-caching {
   // request to an external service (DockerHub, GCR).  If a call fails to lookup a
   // Docker hash, it will fail.
   lookup-docker-hash = false
+
+  // This option determines whether file contents are hashed to determine cache hit or miss.
+  hash-file-contents = true
+  // This option determines whether file paths are hashed to determine cache hit or miss.
+  hash-file-paths = false
```

Comment:
I believe the argument (at least for docker hashing) was that the simple case was so simple we might as well do it anyways. It'd also enable a case where e.g. I write a job's hashes with both and then later am running with only the docker name, I'd still be able to get a cache hit

I could see a similar argument for the file thing